src/ooxml-viewer.ts

```diff
@@ -389,26 +394,28 @@ export class OOXMLViewer {
    */
   private async reformatOpenTabs(): Promise<void> {
     try {
-      const filePathsInOoxmlPackage = new Set(Object.keys(this.zip.files));
-      workspace.textDocuments
-        .filter(d => this.cache.pathBelongsToCache(d.fileName))
-        .map(d => this.cache.getFilePathFromCacheFilePath(d.fileName))
-        .filter(p => filePathsInOoxmlPackage.has(p))
-        .forEach(async filePath => {
-          try {
-            await this.tryFormatXml(filePath);
-          } catch (err) {
-            console.error(err);
-          }
-        });
-
-      workspace.textDocuments
-        .filter(d => this.cache.pathBelongsToCache(d.fileName))
-        .filter(d => !filePathsInOoxmlPackage.has(this.cache.getFilePathFromCacheFilePath(d.fileName)))
-        .forEach(async d => {
-          await window.showTextDocument(Uri.file(d.fileName), { preview: true, preserveFocus: false });
-          await commands.executeCommand('workbench.action.closeActiveEditor');
-        });
+      if (workspace.textDocuments.length) {
```

Comment:
Do we need this if check?

Reply:
I put it there to avoid an error in the "closeWatchers should call restore on the array of file system watchers" test, but refactoring to fix the other tests has made it unnecessary and the test passes without it now, so I removed this if check.